content/en/docs/tasks/security/authz-http/index.md

```diff
@@ -32,178 +30,243 @@ the product page, you can see the following sections:
-`productpage`, `details`, `reviews`, or `ratings`
-is accessible by services in the same namespace (i.e., `default`) and services in the `istio-system` namespace.
-
-Run the following command to create a namespace-level access control policy:
-
-{{< text_import file="examples/TestAuthorizationForHTTPServices.txt" snippet="enforcing_namespace_level_access_control_apply.sh" >}}
+## Enforce mesh-level access control
 
-Once applied, the policy has the following effects:
+Using Istio, you can easily setup mesh-level access control for all workloads in your mesh.
 
-*   Creates a `ServiceRole` `service-viewer` which allows read access to any service in the `default` namespace that has
-the `app` label
-set to one of the values `productpage`, `details`, `reviews`, or `ratings`. Note that there is a
-constraint specifying that
-the services must have one of the listed `app` labels.
+Run the following command to create a mesh-level access control policy `deny-all` policy
+with no workload selector in the root namespace `istio-system`, the policy will select all
+workloads in the mesh:
 
-    {{< text_import file="examples/TestAuthorizationForHTTPServices.txt" snippet="enforcing_namespace_level_access_control_service_viewer.yaml" >}}
-
-*   Creates a `ServiceRoleBinding` that assigns the `service-viewer` role to all services in the `istio-system` and `default` namespaces.
-
-    {{< text_import file="examples/TestAuthorizationForHTTPServices.txt" snippet="enforcing_namespace_level_access_control_bind_service_viewer.yaml" >}}
-
-You can expect to see output similar to the following:
-
-{{< text plain >}}
-servicerole "service-viewer" created
-servicerolebinding "bind-service-viewer" created
+{{< text bash >}}
+$ kubectl apply -f - <<EOF
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: deny-all
+  namespace: istio-system
+spec:
+  {}
+EOF
 {{< /text >}}
 
-Now if you point your browser at Bookinfo's `productpage` (`http://$GATEWAY_URL/productpage`). You should see the "Bookinfo Sample" page,
-with the "Book Details" section in the lower left part and the "Book Reviews" section in the lower right part.
-
-{{< tip >}}
-There may be some delays due to caching and other propagation overhead.
-{{< /tip >}}
+Point your browser at the Bookinfo `productpage` (`http://$GATEWAY_URL/productpage`).
+Now you should see `"RBAC: access denied"`. The error shows that the configured `deny-all`
+policy is working as intended, and Istio doesn't have any rules that allow any access to
+workloads in the mesh.
 
-### Cleanup namespace-level access control
+### Disable mesh-level access control
 
-Remove the following configuration before you proceed to the next task:
+Disable the configured access control with the following command before continuing:
 
 {{< text bash >}}
-$ kubectl delete -f @samples/bookinfo/platform/kube/rbac/namespace-policy.yaml@
+$ kubectl delete authorizationpolicy.security.istio.io/deny-all -n istio-system
```

Comment:
[nit] just `authorizationpolicy` may be sufficient.

Reply:
This is to avoid a tricky case where user is upgrading from 1.3 where there is a `authorizationpolicy.istio.io` resource for some legacy reason. That CRD is never used though.